src/Utilities/FlowAnalysis/Analysis/TaintedDataAnalysis/DllSinks.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using Analyzer.Utilities.Extensions;
+
+namespace Analyzer.Utilities.FlowAnalysis.Analysis.TaintedDataAnalysis
+{
+    internal static class DllSinks
+    {
+        /// <summary>
+        /// <see cref="SinkInfo"/>s for tainted data Dll sinks.
+        /// </summary>
+        public static ImmutableHashSet<SinkInfo> SinkInfos { get; }
+
+        static DllSinks()
+        {
+            ImmutableHashSet<SinkInfo>.Builder sinkInfosBuilder = ImmutableHashSet.CreateBuilder<SinkInfo>();
+
+            sinkInfosBuilder.AddSink(
```

Comment:
>sinkInfosBuilder.AddSink [](start = 12, length = 24)

The more I see this, the more I feel you deserve to put these in a configuration file. Let me stop by your office later today to go over the configuration design.